### ISSUE TITLE
Is the data correct?

### ISSUE THREAD

**Issue by 0xnomy** [fahad-mirza/pakistan_census_2023_tables#1]
Hey Fahd,

I've been trying to work with your data, but I noticed that the numbers don't seem to match the original data available on the PBS website. I consolidated all your files for table 1, and when I calculated the sum (first including "overall" and then removing it), the totals don't align with the national table provided on the PBS website. Specifically, in the "all sex" category, your consolidated dataset sums to 487 million, while the actual value in the PBS PDF is 241 million.

Could you please look into this?

**Comment by fahad-mirza** [comment]
Seems to be coming out fine for me when I sum up the values. There is a small issue in the first table where a tehsil is counted as district due to the way table was processed but if you remove double counts then it is correct. You need to add the column "all_sexes" where region is "OVERALL"

**Comment by fahad-mirza** [comment]
I will adjust the tables to exclude the tehsil level observation being counted as district but overall and mostly, the data is fine

**Comment by fahad-mirza** [comment]
Thanks for starting the issue

**Comment by 0xnomy** [comment]
No worries, would it be possible for you to proivde me a 1 master file for a specific tablet that has data from all provinces of the table?

**Comment by 0xnomy** [comment]
I'm having difficulty creating a dashboard from this dataset. 

When I filter the entire dataset for table 1 by "overall" and sum up the "all_sexes" column, I get approximately 497 million, which is far from the original figure stated on PBS.

![image](https://github.com/user-attachments/assets/3412bcd6-cd0a-4921-b841-9a22faff0f26)

Please let me know if I'm doing anything wrong here. 

I'm; 
- making on masterfile for each table
- by adding a province column and conolsidating all the values from rest of the table i.e table01_balochistan_all etc
- summing the all_sex values  = 497M = wrong
- unpivoting the all_sex value column for better visuals still getting the same value.

**Comment by fahad-mirza** [comment]
Right, so the data ending with the name "_all" contains province, district and tehsil numbers. Summing it up without adding a condition of selecting admin_unit will double or even triple count the data. 

If you use Stata by any chance then by using the command in the Balochistan table "Table_01_Balochistan_all":
`tabstat all_sexes if region == "OVERALL" & admin_unit == "DISTRICT", stat(sum) format(%13.0f) by(name_admin_unit)`

will show you:
![image](https://github.com/user-attachments/assets/0119780e-5286-4781-b2f2-8fd31bf9785f)

Which sums up the total population of Balochistan by district. So just use another filter for admin_unit as you want to count each type of unit uniquely.

**Comment by fahad-mirza** [comment]
The error I found appears in the Sindh data, where Tando Muhammad Khan is being counted twice as a district. This is because Tando Muhammad Khan is both a district and tehsil. So in the district level data, if you remove Tando Muhammad Khan that has the population = 272427, then that will correct this.

There is a chance this is happening in other tables and if you find such double counting of districts then just compare number with original table and remove the one that is erroneously captured.

**Comment by fahad-mirza** [comment]
I will try and correct this bit in the data when i get time and update these datasets. other than that the numbers seem to be alright mainly. But do let me know if you come across other issues as this was a big process.

**Comment by 0xnomy** [comment]
I figured it out, thanks for your help!
